python/src/wslink/websocket.py

```diff
@@ -285,7 +290,7 @@ def sendWrappedMessage(self, rpcid, content, method=''):
         wrapper = {
             "wslink": "1.0",
             "id": rpcid,
-            "result": content,
+            "result": content if content else [],
```

Comment:
The test failures are just looking for specific return values. But I'd like to understand why a return value is required.